src/util.js

```diff
@@ -20,6 +26,14 @@ function combineLinters( results ) {
 		}, {} );
 }
 
+/**
+ * Create a GitHub Gist for displaying messages in the gitHub UI.
```

Comment:
The function's a bit more generic:

```suggestion
 * Create a GitHub Gist (used for logging raw lint data).
```